packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.ts

```diff
@@ -451,13 +455,11 @@ export function useShopperBasketsMutation<Action extends ShopperBasketsMutationT
     return useMutation<Data, Error, Params>(
         (params, apiClients) => {
             const method = apiClients['shopperBaskets'][action] as MutationFunction<Data, Params>
-            return (
-                method.call as (
-                    apiClient: ShopperBasketsClient,
-                    params: Params,
-                    rawResponse: boolean | undefined
-                ) => any
-            )(apiClients['shopperBaskets'], {...params, headers}, rawResponse)
+            return (method.call as (
+                apiClient: ShopperBasketsClient,
+                params: Params,
+                rawResponse: boolean | undefined
+            ) => any)(apiClients['shopperBaskets'], {...params, headers}, rawResponse)
```

Comment:
Has anything changed here, or was this just a formatting change?

Reply:
This is a format change. 🤔